diesel/src/sqlite/connection/mod.rs

```diff
@@ -54,6 +54,10 @@ impl Connection for SqliteConnection {
     type Backend = Sqlite;
     type TransactionManager = AnsiTransactionManager;
 
+    /// Establish a connection to `database_url`.
+    ///
+    /// If the file `database_url` does not exist, this method will
```

Comment:
Do I not understand this or isn't the `database_url` not a file and in the `establish` function aren't we trying to establish a connection to the `database_url` and that the `establish` function has nothing to do with files?

Reply:
SQLite3 databases are files,  its database_url are `file://` . I can remove the word `file` as I don't want to distract readers from the main point of this doc comment.

Reply:
@keepsimple1 Yes, that would be great. Thanks for the clarification!